contrib/fine-tuning/upload-hf.sh

```diff
@@ -20,12 +20,12 @@ hf upload --quiet --repo-type dataset CycloneDX/cdx-docs ./guides guides
 hf upload --quiet --repo-type dataset CycloneDX/cdx-docs ./semantics semantics
 
 echo "Uploading models. Please wait ..."
-hf upload --quiet --repo-type model ${QUANT_MODEL_8BIT} ./${QUANT_MODEL_8BIT} .
-hf upload --quiet --repo-type model ${QUANT_MODEL_6BIT} ./${QUANT_MODEL_6BIT} .
+hf upload --quiet --exclude "**/README.md" --repo-type model ${QUANT_MODEL_8BIT} ./${QUANT_MODEL_8BIT} .
```

Comment:
Our fine-tuning operation was creating a default README page, which was overwriting the full README with the various technical reports on Hugging Face. I have to edit the README page on HF every time there is a new upload 😒. Perhaps, I need to find a place to store the HF README pages.